nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -75,6 +75,7 @@ def next_event(self):
 
 class TelioLoggerCbImpl(libtelio.TelioLoggerCb):
     def log(self, log_level, payload):
+        # print(f"{datetime.datetime.now()} {log_level} {payload}", flush=True)
```

Comment:
To be honest I left it here deliberately as I found this super useful when coding - this way I could see all `telio_*_log!()` macro statements in the natlab run. I just commented it out so it would be there. Any other way to achieve this?

Reply:
Could this be a optional behaviour controlled by a flag in `run_local.py` ?